File: exercises/beecrowd/1097/1097.go
// https://judge.beecrowd.com/en/problems/view/1097
package main

import "fmt"

func main() {
	for i := 1; i <= 9; i += 2 {
		j := i + 6
		for k := 0; k < 3; k++ {
			fmt.Printf("I=%d J=%d\n", i, j-k)
		}
	}
}

File: exercises/beecrowd/1097/1097.ts
// https://judge.beecrowd.com/en/problems/view/1097
function main(): void {
  for (let i = 1; i <= 9; i += 2) {
    let j = i + 6;
    for (let k = 0; k < 3; k++) {
      console.log(`I=${i} J=${j - k}`);
    }
  }
}

main();
